src/NzbDrone.Core/Profiles/Releases/PreferredWordService.cs

```diff
@@ -3,14 +3,15 @@
 using System.Collections.Generic;
 using System.Linq;
 using NzbDrone.Common.Extensions;
+using System;
 
 namespace NzbDrone.Core.Profiles.Releases
 {
     public interface IPreferredWordService
     {
         int Calculate(Series series, string title, int indexerId);
-        List<string> GetMatchingPreferredWords(Series series, string title);
-    }
+        Dictionary<string, List<string>> GetMatchingPreferredWordsGroupByProfile(Series series, string title);
```

Comment:
The original name is still accurate, it returns both grouped and ungrouped values.

Reply:
Reverted to the original name, and returned the new result class.